docs/sources/panels/field-configuration-options.md

```diff
@@ -42,26 +43,19 @@ Standard field options are:
 - [Value mappings](#value-mapping)
 - [Data links](#data-links)
 
-You can apply standard field options to the following panel visualizations:
-
-- [Bar gauge]({{< relref "visualizations/bar-gauge-panel.md" >}})
-- [Gauge]({{< relref "visualizations/gauge-panel.md" >}})
-- [Stat]({{< relref "visualizations/stat-panel.md" >}})
-- [Table]({{< relref "visualizations/table-panel.md" >}})
+You can apply standard field options to most of the built-in Grafana panels. Some older panels and community panels that have
```

Comment:
The standard options apply to every visualization -- for the built in visualizations everything shown is used.  For older/community panels it really depends on how they implemented it.

Reply:
So instead of listing all core visualizations (except heatmap) I figured it's better to explain why core panels use has field options and some don't.